src/singlelayerqg.jl

```diff
@@ -106,7 +106,13 @@ function Problem(dev::Device=CPU();
   # topographic PV
   eta === nothing && (eta = zeros(dev, T, (nx, ny)))
 
-  params = deformation_radius == Inf ? BarotropicQGParams(grid, T(β), T(U), eta, T(μ), T(ν), nν, calcF) : EquivalentBarotropicQGParams(grid, T(β), T(deformation_radius), T(U), eta, T(μ), T(ν), nν, calcF)
+  if U isa Number
+	U = T(U)
```

Comment:
```suggestion
	U = convert(T, U)
```

just a little thing to help interpretability